scripts/internal/get-security-hub-findings/main.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+	"github.com/aws/aws-sdk-go-v2/service/securityhub"
+	"github.com/aws/aws-sdk-go-v2/service/securityhub/types"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"log"
+	"os"
+)
+
+func getSecretsManagerSecret(cfg aws.Config, secretName string) string {
+	client := secretsmanager.NewFromConfig(cfg)
+	input := &secretsmanager.GetSecretValueInput{
+		SecretId:     aws.String(secretName),
+		VersionStage: aws.String("AWSCURRENT"),
+	}
+	result, err := client.GetSecretValue(context.TODO(), input)
+	if err != nil {
+		log.Print(err)
+	}
+	return *result.SecretString
+}
+
+func getMPAccounts(cfg aws.Config) map[string]string {
+	accounts := make(map[string]string)
+	// Get accounts secret
+	environments := getSecretsManagerSecret(cfg, "environment_management")
+
+	var allAccounts map[string]interface{}
+	json.Unmarshal([]byte(environments), &allAccounts)
+
+	for _, record := range allAccounts {
+		if rec, ok := record.(map[string]interface{}); ok {
+			for key, val := range rec {
+				// if strings.Contains(key, "sprinkler-development") {
```

Comment:
I would remove the commented-out code

Reply:
I would to, but I think if I do I'll forget that I can filter it to one account easily here 😂